lib/customer.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'takeaway'
+
+class Customer < Takeaway
+  
```

Comment:
Is Customer a Takeaway?

Inheritance is dangerous - and this is definitely not a good use case.